tokio-console/src/config.rs

```diff
@@ -156,6 +188,44 @@ impl KnownWarnings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub(crate) enum AllowedWarnings {
+    All,
+    Some(BTreeSet<KnownWarnings>),
```

Comment:
I think `Some` here might be confusing, because it's similar to `Option::Some`. Could we call this option `Explicit` or `Set` or something like that instead?

Reply:
Updated. Nice point!